galactic/release-rhel-build.yaml

```diff
@@ -17,7 +17,7 @@ notifications:
 package_blacklist:
 - acado_vendor  # Requires unreleased changes
 - backward_ros  # Not yet generated for RHEL
-- cartographer  # RPM for ceres-solver is not yet stable
+- cartographer  # Build failures on RHEL
```

Comment:
Just to double check. Cartographer was dropped from the other two exclusion lists because it's building there but there are still build failures for the version in Galactic? Or is it a Focal vs Jammy thing?